.github/workflows/_ci.yml

```diff
@@ -193,6 +207,17 @@ jobs:
         echo "disk space at end of build:"
         df -h
         echo
-        du -ch "${{ runner.temp }}" | grep -E "[0-9]{2,}M|[0-9]G"
+        if [[ "${{ inputs.run-du }}" != "false" ]]; then
+            du -ch "${{ inputs.temp-dir || runner.temp }}" | grep -E "[0-9]{2,}M|[0-9]G"
+        fi
       name: "Check disk space at end"
       shell: bash
+<<<<<<< HEAD
```

Comment:
remove

Reply:
argh - i pushed to the wrong pr - ill cleanup